### ISSUE TITLE
product image upload issue 

### ISSUE THREAD

**Issue by senrysaRanjeet** [thaiphan/magento2-s3#21]
every thing has been configured but when i will upload any product image in s3 i will get the file content as like below:

what was this how i will configured it 

0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 nextdoorhub [15/Nov/2017:06:37:49 +0000] 111.93.168.150 0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 2147406AC1F35F66 REST.HEAD.BUCKET - "HEAD /nextdoorhub HTTP/1.1" 200 - - - 16 15 "-" "S3Console/0.4, aws-internal/3" -
0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 nextdoorhub [15/Nov/2017:06:37:49 +0000] 111.93.168.150 0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 A2BCA2C6FD76E003 REST.GET.VERSIONING - "GET /nextdoorhub?versioning= HTTP/1.1" 200 - 162 - 7 - "-" "S3Console/0.4, aws-internal/3" -

**Comment by thaiphan** [comment]
Hey @senrysaRanjeet,

I'm not sure where you're getting this error. Can you post a screenshot?

Regards,

Thai

**Comment by senrysaRanjeet** [comment]
we have configured this repository but when we try to open our media file in amazon s3 the file content is like below:


0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 nextdoorhub [15/Nov/2017:06:37:49 +0000] 111.93.168.150 0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 2147406AC1F35F66 REST.HEAD.BUCKET - "HEAD /nextdoorhub HTTP/1.1" 200 - - - 16 15 "-" "S3Console/0.4, aws-internal/3" -
0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 nextdoorhub [15/Nov/2017:06:37:49 +0000] 111.93.168.150 0fb2c48877f209ff5adf2017a08d6ad947d032067f56a54fcc0611757fe5ccd5 A2BCA2C6FD76E003 REST.GET.VERSIONING - "GET /nextdoorhub?versioning= HTTP/1.1" 200 - 162 - 7 - "-" "S3Console/0.4, aws-internal/3" -

**Comment by thaiphan** [comment]
How are you uploading your media file? Can you write out what you do?

**Comment by senrysaRanjeet** [comment]
Hi,
Catalog image has been uploaded into s3 and i have enabled the extension but after clearing the magento cache i didn't get the images from s3.
what was the issue and as per you what i have to change it there ?

thansk

**Comment by thaiphan** [comment]
Hi @senrysaRanjeet,

Are you planning on setting a custom URL for your media paths or are you planning on routing everything through get.php?

Regards,

Thai

**Comment by senrysaRanjeet** [comment]
Hi,
i just want to show all the media images from s3 server, all the images has been exported into amazon s3 but not showing yet in my magento shop. what i do for it ?

Thanks

**Comment by senrysaRanjeet** [comment]
Hi, 
The media file is showing into my store and it's source is from s3. but i have one question regarding product upload.

1. When a product has been added into catalog and added a product image on it, the particular image has been upload into local server path and amazon s3 also at a time .

so, i want to upload the file only on s3 not in local server, there is any way to mange it.

Thanks 
please response

**Comment by thaiphan** [comment]
Hi @senrysaRanjeet,

> so, i want to upload the file only on s3 not in local server, there is any way to mange it.

Sorry, Magento is just built that way! When I made this extension, I figured that the files on the local server would get deleted as you spin up and delete servers in Elastic Beanstalk or similar setup.

It should be safe to delete the images off your local server **(although always remember to backup just in case)**.

I hope you enjoy using my extension!

Regards,

Thai
